TF iOS SDK (#2081)
saucelabs/sauce-docs: docs/test-fairy/sdk/ios/hiding-webview.md

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

> TestFairy enables developers to hide specific HTML elements from a recorded video in any `UIWebView` or `WKWebView` within their application. As the developer, you may hide one or more elements from the video for security and privacy reasons.

Author: ```suggestion
TestFairy enables developers to hide specific HTML elements from a recorded video in any `UIWebView` or `WKWebView` in their application. As the developer, you may hide one or more elements from the video for security and privacy reasons.
```

> For example, you should prevent all information related to credit card data from appearing in the session.

Author: ```suggestion
For example, you may prevent all information related to credit card data from appearing in the session.
```

## Syntax

To hide an element from video, all you need to do is call the static instance method `hideWebViewElements` in the `TestFairy` class, passing in a CSS selector to target your element:

```css
[TestFairy hideWebViewElements:@"body"];
[TestFairy hideWebViewElements:@"div#username,div#password"];
[TestFairy hideWebViewElements:@".col-12,h1 .header"];
```

> TestFairy will find any `UIWebView` or `WKWebView` in the view hierarchy and hide a given HTML element based on a valid CSS selector.

Author: ```suggestion
TestFairy finds any `UIWebView` or `WKWebView` in the view hierarchy and hides a given HTML element based on a valid CSS selector.
```

> ### Sample video

Author: ```suggestion
### Example
```
